de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/system/40-sw01_swat/5445140_sys_FRESH-203_Doc_BaseType_And_SubType_MailCOnfig.sql

```diff
@@ -0,0 +1,186 @@
+;
+
+-- 05.05.2016 17:02
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+ALTER TABLE AD_MailConfig ADD DocSubType VARCHAR(25) DEFAULT NULL 
+;
+
+-- 05.05.2016 17:21
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Column SET EntityType='de.metas.swat',Updated=TO_TIMESTAMP('2016-05-05 17:21:06','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=544219
+;
+
+commit;
+-- 05.05.2016 17:21
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO t_alter_column values('ad_mailconfig','AD_MailBox_ID','NUMERIC(10)',null,null)
+;
+
+commit;-- 05.05.2016 17:48
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Table SET AD_Window_ID=540078,Updated=TO_TIMESTAMP('2016-05-05 17:48:30','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Table_ID=540241
+;
+
+-- 05.05.2016 17:49
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Tab SET EntityType='de.metas.swat',Updated=TO_TIMESTAMP('2016-05-05 17:49:28','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Tab_ID=540231
+;
+
+-- 05.05.2016 17:49
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Field (AD_Client_ID,AD_Column_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,Created,CreatedBy,Description,DisplayLength,EntityType,Help,IsActive,IsCentrallyMaintained,IsDisplayed,IsEncrypted,IsFieldOnly,IsHeading,IsReadOnly,IsSameLine,Name,Updated,UpdatedBy) VALUES (0,554392,556927,0,540231,TO_TIMESTAMP('2016-05-05 17:49:48','YYYY-MM-DD HH24:MI:SS'),100,'Logical type of document',25,'de.metas.swat','The Document Base Type identifies the base or starting point for a document.  Multiple document types may share a single document base type.','Y','Y','Y','N','N','N','N','N','Document BaseType',TO_TIMESTAMP('2016-05-05 17:49:48','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 05.05.2016 17:49
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Field_Trl (AD_Language,AD_Field_ID, Description,Help,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language,t.AD_Field_ID, t.Description,t.Help,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Field t WHERE l.IsActive='Y' AND l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N' AND t.AD_Field_ID=556927 AND NOT EXISTS (SELECT * FROM AD_Field_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Field_ID=t.AD_Field_ID)
+;
+
+-- 05.05.2016 17:50
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
```

Comment:
For this field, i set IsCentrallyMaintained='N', so as far as the UI user in concerned, your documentation will be preserved.